evap/staff/tests/test_importers.py

```diff
@@ -341,7 +341,8 @@ def create_existing_course(self) -> tuple[Course, Evaluation]:
         return existing_course, existing_course_evaluation
 
     def test_valid_file_import(self):
-        importer_log = import_enrollments(self.default_excel_content, self.semester, None, None, test_run=True)
+        with assert_no_database_useless_logging():
+            importer_log = import_enrollments(self.default_excel_content, self.semester, None, None, test_run=True)
```

Comment:
My understanding was that this doesn't do any actual database modifications, but it just "accidentally" added the empty log entries. If this PR fixes the empty log entries, we should be good to use `assert_no_database_modifications` (and hopefully we don't need `assert_no_database_useless_logging` at all, then)